index.js

```diff
@@ -0,0 +1,23 @@
+// import { DateTime } from "./node_modules/luxon/build/es6/luxon.js";
+// console.log(DateTime)
+
+import DisplayBook from './modules/displayBook.js';
```

Comment:
- [ ] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.